meetings/2023-07/july-11.md

```diff
@@ -160,7 +160,7 @@ Congratulations: Standards approved by GA 27 June and posted on website:
 - For any editorial changes to the two approved standards, please advise ASAP to the Ecma Secretariat (Samina Husain, Patrick Charolais, Allen Wirfs-Brock, Istvan Sebestyen).
 - New TC39 Member Oramasearch approved, announced and welcomed.
 - For the TC39 meeting contributions, the summary notes from each contributor are very relevant and are requested to be added to the meeting notes.  These summary notes ensure accurate meeting minutes.
-- For the  ES2023 «Nice PDF» Versions next steps will be taken to find a solution for 2024.
+- For the ES2023 «Nice PDF» Versions next steps will be taken to find a solution for 2024.
```

Comment:
this is interesting.. I was puzzled because I couldn't understand why this was showing up in the diff, when it was already fixed in a previous commit.  Well, it wasn't.  This duplicates line 141 (where this got fixed).  In the fix code, it skips over anything the lexer does not consider a paragraph, so in this case a list.  The reason the other one was fixed was because the list was not formatted correctly and so is detected as a paragraph.  See screenshot of parsed html version displayed by GH below.  All of this is basically to say, we should add coverage for fixing these problems in list blocks as well.  Of course the surface area for these problems is much smaller, so it's not urgent.

![Screenshot 2023-09-10 at 10 05 41 AM](https://github.com/tc39/notes/assets/15326526/af538057-89dd-4a1a-b13b-62fd174443e9)